### PR TITLE
--no-cache-dir

### PR DESCRIPTION
**Pull request by elduddz**

### CODE REVIEW

File: linux/aws/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 FROM python:3.6-slim
 
-RUN pip3 install --upgrade awscli==1.16.291
+RUN pip3 install --upgrade awscli==1.16.291 --no-cache-dir
```
Comment (elduddz) [inline linux/aws/Dockerfile:3]:
```suggestion
RUN pip3 install --no-cache-dir --upgrade awscli==1.16.291
```

**Comment by eganjs** [comment]
Looks like `linux/awscli/Dockerfile`needs this fix too

**Comment by elduddz** [comment]
> Looks like `linux/awscli/Dockerfile`needs this fix too

Yer

**Review by eganjs — APPROVED** [review]

**Review by eganjs — APPROVED** [review]

File: linux-archive/jdk8-python36/Dockerfile [hunk 2]
```diff
@@ -13,14 +13,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     python3.6=* \
     unzip=* \
     wget=* \
-    python3.6-dev=* \
```
Comment (eganjs) [inline linux-archive/jdk8-python36/Dockerfile:16]: Can you restore this line

**Review by eganjs — CHANGES_REQUESTED** [review]

File: linux-archive/jdk8-python36/Dockerfile (hunk 2 above)
Comment (elduddz) [reply linux-archive/jdk8-python36/Dockerfile:16]: ok put back

**Review by eganjs — APPROVED** [review]

**Pull request merged**